grails-app/controllers/org/pih/warehouse/api/LocalizationApiController.groovy

```diff
@@ -21,30 +21,42 @@ class LocalizationApiController {
     GrailsApplication grailsApplication
 
     def list() {
-        String languageCode = params.lang
+        String languageCode = params.languageCode
         String prefix = params.prefix
 
         String[] supportedLocales = grailsApplication.config.openboxes.locale.supportedLocales
 
+        Locale defaultLocale = Locale.default
```

Comment:
How do you feel about moving this to LocalizationService.getDefaultLocale()?
```
LocalizationService.getDefaultLocale() {
    return Locale.default
```
That way if we want to change the behavior we can do that in one place. For example I think the default locale should actually be the one we specify in the config, not what the system thinks. 

By the way, we should document and figure out the different ways to set the default locale and what the precedence should be. It might be the case that the Application config should actually just use the JVM argument or system locale by default. Probably worth a discovery ticket.

* System default locale (sudo update-locale LANG=fr_FR.UTF-8)
* JVM argument (-Duser.language=en -Duser.region=US)
* Application Config [application.yml] (locale.defaultLocale)
* Application Config override [openboxes.yml] (local.defaultLocale)

Reply:
If it works the way it did, then no change is required. 

And I agree that we should consolidate the additional scope we've identified, add into a ticket or tickets and work on them based on the trigger points I mentioned in one of the other comments.